demo/index.html

```diff
@@ -58,21 +57,20 @@
           <div class="progress-value"><span>{{ratio}}</span>%</div>
         </div>
       </template>
-    </dom-module>
-
-    <script>
-      HTMLImports.whenReady(function() {
-        Polymer({
-          is: 'x-progressbar',
+      <script>
+        HTMLImports.whenReady(function() {
```

Comment:
Actually, on second thought, not a 🚲🏠  at all, this won't work without `WebComponentsReady`